build.sbt

```diff
@@ -3,4 +3,18 @@ version := "0.1"
 scalaVersion := "2.12.4"
 
 // https://mvnrepository.com/artifact/uk.co.jhc/sqlest
-libraryDependencies += "uk.co.jhc" %% "sqlest" % "0.8.10"
+libraryDependencies ++= Seq(
+  "com.typesafe" % "config" % "1.3.0",
+  "uk.co.jhc" %% "sqlest" % "0.8.8",
+  "org.json4s" %% "json4s-native" % "3.5.1",
+  "org.json4s" %% "json4s-ext" % "3.5.1",  
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
+  "org.scalatest" %% "scalatest" % "3.0.1" % "test",
+  "org.scalactic" %% "scalactic" % "3.0.1",
+  "com.h2database" % "h2" % "1.4.191",
+  "org.http4s" %% "http4s-dsl" % "0.15.8",
+  "org.http4s" %% "http4s-blaze-server" % "0.15.8"
```

Comment:
The only ones we need right now are the com.typesafe config one, the sqlest one, and the h2database one. I'd ditch the rest.

Reply:
Done